pkgs/servers/web-apps/outline/default.nix

```diff
@@ -0,0 +1,88 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, nodejs
+, yarn
+, yarn2nix-moretea
+}: let
```

Comment:
```suggestion
}:

let
```
Why is this a let in? This can be moved into mkDerivation.

Reply:
Oh yeah, that's a leftover from playing around with Yarn - you're right